assignments/lambda-classes.js

```diff
@@ -1 +1,265 @@
 // CODE here for your Lambda Classes
+
+// Person class (base class)
+
+class Person {
+
+  constructor(props) {
+
+    this.name = props.name;
+    this.age = props.age;
+    this.gender = props.gender;
+    this.location = props.location;
```

Comment:
You could use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) here:
```javascript
constructor({ name, age, gender, location }) {
    this.name = name;
    this.age = age;
    this.gender = gender;
    this.location = location;
}
```
You'll be seeing this more as you get into [React.js](https://reactjs.org/), so learning it now would be a good way to get ahead with some of the syntax.